pkg/helm/actions/rollback_release_test.go

```diff
@@ -0,0 +1,83 @@
+package actions
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chartutil"
+	kubefake "helm.sh/helm/v3/pkg/kube/fake"
+	"helm.sh/helm/v3/pkg/release"
+	"helm.sh/helm/v3/pkg/storage"
+	"helm.sh/helm/v3/pkg/storage/driver"
+)
+
+func TestRollbackRelease(t *testing.T) {
+	tests := []struct {
+		name        string
+		releaseName string
+		release     release.Release
+	}{
+		{
+			name: "valid release rollback",
+			release: release.Release{
+				Version: 1,
+				Name:    "valid-release",
+				Info: &release.Info{
+					Status: release.StatusDeployed,
+				},
+			},
+		},
+		{
+			name: "invalid release rollback",
+			release: release.Release{
+				Name:    "invalid-release",
+				Version: 1,
+				Info: &release.Info{
+					Status: release.StatusDeployed,
+				},
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		store := storage.Init(driver.NewMemory())
+		t.Run(tt.name, func(t *testing.T) {
+			// create fake release
+			actionConfig := &action.Configuration{
+				Releases:     store,
+				KubeClient:   &kubefake.PrintingKubeClient{Out: ioutil.Discard},
+				Capabilities: chartutil.DefaultCapabilities,
+				Log:          func(format string, v ...interface{}) {},
+			}
+			if tt.name == "invalid release rollback" {
```

Comment:
see above - we should simplify the test logic - do not implement test fixture base on its name.

Reply:
Done